Hi @bondada-a !
Thank you for finding this problem. I can confirm that after upgrading humble packages to the newest version this problem arises:

```bash
# ...
Starting >>> robotiq_hande_driver
--- stderr: robotiq_hande_driver                             
/tmp/ros_ws_tmp2/src/robotiq_hande_driver/robotiq_hande_driver/test/test_load_hw_interface.cpp:12:10: fatal error: ros2_control_test_assets/components_urdfs.hpp: No such file or directory
   12 | #include "ros2_control_test_assets/components_urdfs.hpp"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

--- a/robotiq_hande_driver/CHANGELOG.md
+++ b/robotiq_hande_driver/CHANGELOG.md
@@ -12,6 +12,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Deprecated
 ### Removed
 ### Fixed
+* [PR-33](https://github.com/AGH-CEAI/robotiq_hande_driver/pull/33) - Fixed missing ros2_control_test_assets dependency in CMakeLists.txt that causes build failures after ROS2 Humble package updates.
+
 ### Security
 
 ## [0.2.0] - 2025-08-25

